src/nrc/conf/includes/base.py

```diff
@@ -144,326 +27,50 @@
-            "formatter": "performance",
-            "maxBytes": 1024 * 1024 * 10,  # 10 MB
-            "backupCount": 10,
-        },
-        "notifications": {
-            "level": "INFO",
-            "class": "logging.handlers.RotatingFileHandler",
-            "filename": LOGGING_DIR / "notifications.log",
-            "formatter": "performance",
-            "maxBytes": 1024 * 1024 * 10,  # 10 MB
-            "backupCount": 10,
-        },
-        "log_outgoing_requests": {
-            "level": "DEBUG",
-            "formatter": "outgoing_requests",
-            "class": "logging.StreamHandler",  # to write to stdout
-        },
-        "save_outgoing_requests": {
-            "level": "DEBUG",
-            # enabling saving to database
-            "class": "log_outgoing_requests.handlers.DatabaseOutgoingRequestsHandler",
-        },
-    },
-    "loggers": {
-        "": {
-            "handlers": _root_handlers,
-            "level": "ERROR",
-            "propagate": False,
-        },
-        "nrc": {"handlers": _root_handlers, "level": LOG_LEVEL, "propagate": True},
-        "nrc.api.serializers": {
-            "handlers": ["notifications"],
-            "level": LOG_LEVEL,
-            "propagate": False,
-        },
-        "django.request": {
-            "handlers": _django_handlers,
-            "level": LOG_LEVEL,
-            "propagate": True,
-        },
-        "django.template": {
-            "handlers": ["console"],
-            "level": "INFO",
-            "propagate": True,
-        },
-        "mozilla_django_oidc": {
-            "handlers": _root_handlers,
-            "level": LOG_LEVEL,
-        },
-        "vng_api_common": {
-            "handlers": ["console"],
-            "level": LOG_LEVEL,
-            "propagate": True,
-        },
-        "django.db.backends": {
-            "handlers": ["console_db"] if LOG_QUERIES else [],
-            "level": "DEBUG",
-            "propagate": False,
-        },
-        "log_outgoing_requests": {
-            "handlers": (
-                ["log_outgoing_requests", "save_outgoing_requests"]
-                if LOG_REQUESTS
-                else []
-            ),
-            "level": "DEBUG",
-            "propagate": True,
-        },
-    },
+LOGGING["handlers"]["notifications"] = {
+    "level": "INFO",
+    "class": "logging.handlers.RotatingFileHandler",
+    "filename": LOGGING_DIR / "notifications.log",
+    "formatter": "performance",
+    "maxBytes": 1024 * 1024 * 10,  # 10 MB
+    "backupCount": 10,
+}
+LOGGING["loggers"]["nrc.api.serializers"] = {
+    "handlers": ["notifications"],
+    "level": LOG_LEVEL,
+    "propagate": False,
 }
 
-LOG_OUTGOING_REQUESTS_DB_SAVE = config("LOG_OUTGOING_REQUESTS_DB_SAVE", default=False)
+# TODO should be part of OAF?
 LOG_OUTGOING_REQUESTS_DB_SAVE_BODY = config(
     "LOG_OUTGOING_REQUESTS_DB_SAVE_BODY", default=False
 )
+# TODO should be part of OAF?
```

Comment:
I was considering adding this to OAF (it's hardcoded there currently), but the default is `True` there (https://github.com/maykinmedia/open-api-framework/blob/main/open_api_framework/conf/base.py#L653). Do you think it's fine to use default `True` for Open Notificaties as well?